language/de/install.php

```diff
@@ -72,7 +72,7 @@
 
 	<p>phpBB3 unterstützt folgende Datenbank-Typen:</p>
 	<ul>
-		<li>MySQL 3.23 und höher (MySQLi wird unterstützt)</li>
+		<li>MySQL 4.1.3 und höher (MySQLi ist erforderlich)</li>
```

Comment:
Die SUPPORT_BODY Änderung fehlt komplett?

Reply:
Hm anscheinend. Ich dachte ursprünglich, dass das nur Änderungen von 3.2 auf 3.3 in den Links wären, aber auch die hätten ja hier auftauchen müssen.